.github/workflows/test-container-image-tags.yml

```diff
@@ -73,3 +96,23 @@ jobs:
         run: |
           source 'assert.sh'
           assert_eq ${{ steps.container.outputs.image-tags }} "greenbone/actions:unstable" "wrong container tag"
+
+  build-dist:
+    needs: [ build-container-image-tags-action-dist, test-container-image-tags-branch ]
+    runs-on: ubuntu-latest
+    name: "Commit dist of this action to the repo"
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Download dist
+        uses: actions/download-artifact@v2
+        with:
+          name: build
+          path: container-image-tags/dist
+      - name: Commit dist file of this action
+        run: |
+          git config --global user.name "${{ secrets.GREENBONE_BOT }}"
+          git config --global user.email "${{ secrets.GREENBONE_BOT_MAIL }}"
```

Comment:
is this working without setting the remote?

`git remote set-url origin https://${{ secrets.GREENBONE_BOT_TOKEN }}@github.com/${{ github.repository }}`